www/source/partials/docs/_glossary-launcher.html.md.erb

```diff
@@ -0,0 +1,23 @@
+# <a name="glossary-launcher" id="glossary-launcher" data-magellan-target="glossary-launcher">Launcher</a>
+
+Habitat's Launcher is a sidecar process for the Supervisor which provides a mechanism for launching processes on behalf of the Supervisor. It is the entry point for running the Supervisor and is the Supervisor for the Supervisor. Whereas the Supervisor is able to automatically update itself, the Launcher is currently released a bit differently, by design; it should be rare that the Launcher ever needs to change.
+
+To update your Launchers, you will need to run
+
+```
+hab pkg install core/hab-launcher
+```
+
+on each machine that has a Supervisor, and then restart your Supervisor. This will, by necessity, require a restart of supervised services, so factor that into your planning.
+
+The Launcher is designed to run as process 1; it's extremely minimal by design. It's responsibilities are to:
+
+* Launch the Habitat Supervisor
+* Restart the Habitat Supervisor on shutdown (Supervisor for the Supervisor)
+* Start, Stop, and Restart processes on behalf of the Supervisor
+
+Launcher is specifically designed to enable the Supervisor to self update without shutting down or re-parenting the services that the Supervisor is supervising. It is versioned separately from the Supervisor and should be updated very infrequently since an update of the Launcher could require a system restart if it is running as process 1.
```

Comment:
```suggestion
The Launcher enables the Supervisor to update itself without shutting down or re-parenting the Supervisor's services. The Launcher is versioned separately from the Supervisor and should be updated rarely because it requires a system restart.
```
Help clarify: The Launcher runs as process one all of the time, or just some of the time? If all of the time,

Reply:
These aren't the Supervisor's services, they are services that (might) be under supervision. So I don't think this wording adds clarity.

The Launcher might not run as PID 1 in environments like Kubernetes, or if the user installed the Launcher a particular way. If someone is going out of their way to run the Launcher in a particular way, then they would know how they are running it. :)